ProteoformSuiteGUI/DisplayUtility.cs

```diff
@@ -47,21 +47,20 @@ public static void tooltip_graph_display(ToolTip t, MouseEventArgs e, Chart c, P
             }
         }
 
-        public static void GraphRelationsChart(Chart ct, List<ProteoformRelation> relations)
+        public static void GraphRelationsChart(Chart ct, List<ProteoformRelation> relations, string series)
         {
+            ct.Series[series].XValueMember = "group_adjusted_deltaM";
+            ct.Series[series].YValueMembers = "running_sum";
```

Comment:
Running sum/unadjusted group count and group count are slightly different bc group count is the # of relations for a given peak, and the running sum is a more crude count of the number of pairs w/ a delta mass +- 1/2 peak width base (out of range decimal or not).